src/Geosuggest.tsx

```diff
@@ -339,6 +339,7 @@ export default class extends React.Component<IProps, IState> {
           fixturesSearched++;
 
           suggests.push({
+            ...fixture,
```

Comment:
if we go with `...fixture,`, then some of the following lines might be redundant e.g.
`className: fixture.className,` or `label: fixture.label,`

Reply:
@siddiq 👍